### DISCUSSION TITLE
Z-Wave JS Update

### DISCUSSION THREAD

**Discussion by kilian-f-za** [tteck/Proxmox#2818] [Help Needed]
I installed Z-Wave JS via script and already carried out an "update". However, the latest version of Git will not be installed. 

zwave-js-ui: 9.10.3
zwave-js: 12.5.4

There are no errors by the update.
Version 12.5.5 from Zwave-Js is available on Git

How do I get the current version installed?

**Comment by tteck** [comment]
Wait until the new version of node-zwave-js is added to zwave-js-ui.

![Screenshot 2024-04-19 10 10 35 AM](https://github.com/tteck/Proxmox/assets/21342670/826f782c-1db4-4770-b57d-1964272e58b1)
